benefits/cycle_to_work_scheme.md

```diff
@@ -2,7 +2,7 @@
 
 We take part in Cyclescheme as we wish to support those who wish to cycle to work whether it's for pleasure, aiming to be healthier, trying to be more green or any other reasons.
 
-Made Tech have a limit of £2500 with Cyclescheme which can be used to purchase a bike and any accessories you will need. You then pay a monthly 'salary sacrifice' to effectively 'hire' the bike and kit, over a 12 month period. Once this period has been completed you will be given options by Cyclescheme around owning your bike. If you leave Made Tech before paying off the purchase, the balance will be taken from your final salary payment.
+Made Tech use Cyclescheme, who will provide you with an voucher (with no price limit), which can be used to purchase a bike, parts and components, and any accessories you will need. You then pay a monthly 'salary sacrifice' to effectively 'hire' the bike and kit, over a 12 month period. Once this period has been completed you will be given options by Cyclescheme around owning your bike. If you leave Made Tech before paying off the purchase, the balance will be taken from your final salary payment.
```

Comment:
`an voucher` should be  `a voucher`

Reply:
"Options around owning" reads a little vague? Is there any more detail/guidelines on what those are likely to be?

Reply:
@baarkerlounger if I follow Ben's advice below and leave the fine details to cyclescheme we should do the same here. However, it isn't the easiest thing to wrap one's head around.

Reply:
Could maybe link the options directly (https://help.cyclescheme.co.uk/article/42-what-is-an-ownership-fee) here so that at least it's quick to look up? I actually hadn't realised the scheme itself was this complicated...

Reply:
Done and done.